lib/kaur/result_tuple.ex

```diff
@@ -47,7 +67,25 @@ defmodule Kaur.ResultTuple do
   @spec ok?(result_tuple) :: boolean
   def ok?({:ok, _}), do: true
   def ok?({:error, _}), do: false
-  
+
+  @doc """
+  Calls the next function only if we have an error tuple. Otherwise we skip the call and
+  returns the ok tuple
```

Comment:
To prevent the mental hindrance I have with the `or_else` naming (I intuitively want to put several or them and not have them done in succession), it would be useful to indicate in the doc that it is meant to be used only once at the end of a pipeline.